config/config.go

```diff
@@ -886,10 +890,11 @@ func DefaultMempoolConfig() *MempoolConfig {
 		WalPath:   "",
 		// Each signature verification takes .5ms, Size reduced until we implement
 		// ABCI Recheck
-		Size:        5000,
-		MaxTxsBytes: 1024 * 1024 * 1024, // 1GB
-		CacheSize:   10000,
-		MaxTxBytes:  1024 * 1024, // 1MB
+		Size:             5000,
+		MaxTxsBytes:      1024 * 1024 * 1024, // 1GB
+		CacheSize:        10000,
+		MaxTxBytes:       1024 * 1024, // 1MB
+		MaxOutboundPeers: 10,
```

Comment:
```suggestion
		ExperimentalMaxOutboundPeers: 10,
```

Reply:
Changed the name. Does it sound good?
Also changed the default value to 0.